_source/_posts/2022-09-08-react-fastify-postgres.md

```diff
@@ -0,0 +1,938 @@
+---
+layout: blog_post
+title: "How to Build a Secure React and Fastify API App"
+author: gabi-dombrowski
+by: contractor
+communities: [javascript]
+description: "This tutorial demonstrates how to create a complete web application project with security built in from the ground up. You'll create a React frontend and Fastify API backend using TypeScript, use Docker to run PostgreSQL, add authentication to secure the frontend and API, and manage the project in a Lerna monorepo. "
```

Comment:
```suggestion
description: "This tutorial demonstrates how to create a complete web application project with security built in from the ground up. You'll use a React frontend and a Fastify API backend, and manage the project in a Lerna monorepo. "
```
Recommended character count for this field is 200-250 char, so I trimmed it way down.